data/pc/1.20.3/protocol.json

```diff
@@ -0,0 +1,7309 @@
+            }
+          ]
+        ],
+        "packet_kick_disconnect": [
+          "container",
+          [
+            {
+              "name": "reason",
+              "type": "anonymousNbt"
+            }
+          ]
+        ],
+        "packet_profileless_chat": [
+          "container",
+          [
+            {
+              "name": "message",
+              "type": "anonymousNbt"
+            },
+            {
+              "name": "type",
+              "type": "varint"
+            },
+            {
+              "name": "name",
+              "type": "anonymousNbt"
+            },
+            {
+              "name": "target",
+              "type": [
+                "option",
+                "anonymousNbt"
+              ]  
+            }
+          ]
+        ],
+        "packet_entity_status": [
+          "container",
+          [
+            {
+              "name": "entityId",
+              "type": "i32"
+            },
+            {
+              "name": "entityStatus",
+              "type": "i8"
+            }
+          ]
+        ],
+        "packet_explosion": [
+          "container",
+          [
+            {
+              "name": "x",
+              "type": "f64"
+            },
+            {
+              "name": "y",
+              "type": "f64"
+            },
+            {
+              "name": "z",
+              "type": "f64"
+            },
+            {
+              "name": "radius",
+              "type": "f32"
+            },
+            {
+              "name": "affectedBlockOffsets",
+              "type": [
+                "array",
+                {
+                  "countType": "varint",
+                  "type": [
+                    "container",
+                    [
+                      {
+                        "name": "x",
+                        "type": "i8"
+                      },
+                      {
+                        "name": "y",
+                        "type": "i8"
+                      },
+                      {
+                        "name": "z",
+                        "type": "i8"
+                      }
+                    ]
+                  ]
+                }
+              ]
+            },
+            {
+              "name": "playerMotionX",
+              "type": "f32"
+            },
+            {
+              "name": "playerMotionY",
+              "type": "f32"
+            },
+            {
+              "name": "playerMotionZ",
+              "type": "f32"
+            },
+            {
+              "name": "block_interaction_type",
+              "type": "varint"
+            },
+            {
+              "name": "small_explosion_particle",
+              "type": "particle"
+            },
+            {
+              "name": "large_explosion_particle",
+              "type": "particle"
+            },
+            {
+              "name": "explosion_sound_name",
+              "type": "string"
```

Comment:
this seems incorrect, see https://github.com/extremeheat/extracted_minecraft_data/blob/client1.20.3/client/net/minecraft/network/protocol/game/ClientboundExplodePacket.java#L121

Reply:
https://github.com/extremeheat/extracted_minecraft_data/blob/client1.20.3/client/net/minecraft/sounds/SoundEvent.java#L61-L64 

We don't have a soundEvent type so I just wrote what it has out (much like many other types for example the ClientOptions) and its also not used anywhere else that I can see,

Reply:
yeah I see, it is indeed a string, they just parse it further but we don't need to